src/js/themes/base.d.ts

```diff
@@ -1734,11 +1734,16 @@ export interface ThemeType {
       iconOnly?: {
         pad?: PadType;
       };
+      border?: {
```

Comment:
Alphabetize order of properties for code style?

Reply:
Done.